CareKitStore/CareKitStore/CoreData/OCKStore+Outcomes.swift

```diff
@@ -240,6 +240,11 @@ extension OCKStore {
             let beforePredicate = NSPredicate(format: "%K < %@", #keyPath(OCKCDOutcome.date), interval.end as NSDate)
             predicate = NSCompoundPredicate(andPredicateWithSubpredicates: [predicate, afterPredicate, beforePredicate])
         }
+        
+        if !query.uuids.isEmpty {
+            let versionPredicate = NSPredicate(format: "%K IN %@", #keyPath(OCKCDObject.uuid), query.uuids)
+            predicate = NSCompoundPredicate(andPredicateWithSubpredicates: [predicate, versionPredicate])
+        }
```

Comment:
This looks like it might be duplicate code. I see another if statement right below this that also checks if the uuids array is empty.

Reply:
Removed